modules/KoreBuild.Tasks/UpgradeDependencies.cs

```diff
@@ -210,6 +215,12 @@ private int UpdateDependencies(DependencyVersionsFile localVersionsFile, Depende
                     newValue = KoreBuildVersion.Current;
                     Log.LogMessage(MessageImportance.Low, "Setting InternalAspNetCoreSdkPackageVersion to the current version of KoreBuild");
                 }
+                else if (var.Key == "SdkPackageVersion")
```

Comment:
Naming what about: DotNetCliVersion?

We should also add update logic for the "msbuild-sdks" section of the global.json file. See https://github.com/aspnet/BuildTools/tree/dev/src/Microsoft.DotNet.GlobalTools.Sdk#usage for example. We will need this more as we migrated Internal.AspNetCore.Sdk and others away from being a PackageReference and towards being proper MSBuild SDKs.

Also, maybe we should have this information stored separately. Dependencies.props was really only mean for PackageReference versions. Maybe use a new file named Tooling.props or something? Just a thought...haven't committed to the idea.

Reply:
2 should be its own item right? Once I touch 50 repos this is already going to be a messy change, I'd really rather not complicate it further.

Reply:
If by 2 you mean updating the "msbuild-sdks", then no, we need that before we can take this change. The msbuild-sdks section is required in at least 4 repos for the global tool shim stuff I just added yesterday.